README.md

```diff
@@ -6,36 +6,27 @@ This repository contains resources to support the AIDA Interchange Format (AIF).
      This ontology can be validated using the SHACL constraints file in
      `src/main/resources/com/ncc/aif/aida_ontology.shacl`.
 
-*    utilities to make it easier to work with this format.  Java utilities are in
-     `src/main/java/com/ncc/aif/AIFUtils.java`. A Kotlin version of the utilities are in
-	 `src/main/java/edu/isi/gaia/AIFUtils.kt`.  Either of these can be used by adding a Maven dependency on
-     `com.ncc:aida-interchange:1.0.0-SNAPSHOT`.  A Python translation of these utilities
-     is in `python/aida_interchange/aifutils.py`.
+*    utilities to make it easier to work with this format.  Java utilities are
+     in `src/main/java/com/ncc/aif/AIFUtils.java`, which can be used by adding
+     a Maven dependency on `com.ncc:aida-interchange:1.0.0-SNAPSHOT`.  A
```

Comment:
Could you change this to 1.1.0-SNAPSHOT?  We missed that when we updated the version.